src/main/java/com/minecolonies/coremod/entity/ai/citizen/deliveryman/EntityAIWorkDeliveryman.java

```diff
@@ -68,22 +70,17 @@
     /**
      * The inventory's slot which is held in hand.
      */
-    private static final int SLOT_HAND      = 0;
+    private static final int SLOT_HAND = 0;
 
     /**
-     * Next target the deliveryman should gather stuff at.
-     */
-    private BlockPos gatherTarget = null;
-
-    /**
-     * Tracks how many buildings the DMan visited to gather resources.
+     * Warehouse the deliveryman is assigned to.
      */
-    private int gatherCount = 0;
+    private BuildingWareHouse wareHouse = null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "wareHouse" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)